src/Umbraco.Web.UI.Client/src/views/propertyeditors/tags/tags.controller.js

```diff
@@ -142,6 +142,13 @@ angular.module("umbraco")
                 });
             }
 
+            function currentCulture(scope) {
```

Comment:
This is not nice :(  We need to make a directive for this property editor and then the directive can require a `umbTabbedContent` that it can get the `activeVariant` from. We defo shouldn't be recursively looking through parent scopes (i realize this is probably a 'get it done quick thing')